docs/cugraph/source/references/cugraph_ref.md

```diff
@@ -1,97 +1,46 @@
-##########
-References
-##########
+# References
 
-************
-Architecture
-************
+## Architecture
 
 2-D Data Partitioning
 
-- Kang, S., Fender, A., Eaton, J., & Rees, B. (2020, September) *Computing PageRank Scores of Web Crawl Data Using DGX A100 Clusters*. In 2020 IEEE High Performance Extreme Computing Conference (HPEC) (pp. 1-4). IEEE.
+- Kang, S., Fender, A., Eaton, J., & Rees, B. (2020, September) *Computing PageRank Scores of Web Crawl Data Using DGX A100 Clusters*.  In 2020 IEEE High Performance Extreme Computing Conference (HPEC) (pp. 1-4). IEEE.
 
+- S. Kang, J. Nke and B. Rees, (2022 September) *Analyzing Multi-trillion Edge Graphs on Large GPU Clusters: A Case Study with PageRank*, In 2022 IEEE High Performance Extreme Computing Conference (HPEC), Waltham, MA, USA, 2022, pp. 1-7, doi: 10.1109/HPEC55821.2022.9926341.
```

Comment:
Do we want to add the latest paper, or do we need to wait for the conference proceedings to be published?

Reply:
I added the link, readers with IEEE memberships can get the material now, others will have to wait